src/sentry/models/pullrequest.py

```diff
@@ -118,4 +118,4 @@ class PullRequestComment(Model):
     class Meta:
         app_label = "sentry"
         db_table = "sentry_pullrequest_comment"
-        unique_together = ("pull_request", "comment_type")
+        unique_together = (("pull_request", "comment_type"),)
```

Comment:
will this constraint become active once we remove the pull_request constraint? i'm not sure if i need to write some custom SQL operations to make this one the active one

Reply:
No, there are no DDL operations in your migration to add the missing constraint. You could use a SQL operation to define the constraint if it doesn't exist. Using a simple DDL operation could fail if any of the environments/tenants have this constraint already.